### PR TITLE
added manual option to save into localstorage

### PR DESCRIPTION
**Pull request by thmasn**
autosave causes a lot of slowdowns, so i added a manual option to save into localstorage, which is only displayed if autosave is off

### CODE REVIEW

File: editor/js/Menubar.Status.js [hunk 1]
```diff
@@ -18,7 +18,9 @@ Menubar.Status = function ( editor ) {
 		if ( value === true ) {
 
 			editor.signals.sceneGraphChanged.dispatch();
-
+			saveLocal.dom.style.display="none";
```
Comment (GGAlanSmithee) [inline editor/js/Menubar.Status.js]: new line after

**Comment by GGAlanSmithee** [comment]
This seems like a nice addition to me, but you have a lot of syntatical issues with your code, see the [style guide](https://github.com/mrdoob/three.js/wiki/Mr.doob's-Code-Style%E2%84%A2)

**Comment by thmasn** [comment]
i hope i improved that

**Comment by mrdoob** [comment]
Maybe it could be simpler? How about we make it so when we click "autosave" it saves?

**Comment by mrdoob** [comment]
I just tried... Either way is hacky... 😕

**Comment by Mugen87** [comment]
Maybe it makes more sense to figure out a different, more performant kind of autosave? In this way, the need for a manual option will be less important...

**Pull request closed without merging**